assets/src/edit-story/app/story/useStoryReducer/test/combineElements.js

```diff
@@ -47,40 +50,55 @@ describe('combineElements', () => {
     const initial = restore(getDefaultState1());
 
     // Combine non-existing element abc into 789
-    const result = combineElements({ firstId: 'abc', secondId: '789' });
+    const result = combineElements({
+      firstElement: { id: 'abc', type: 'image' },
+      secondId: '789',
+    });
```

Comment:
I don't think this test makes sense anymore. This should actually do a merge and overwrite with new props as long as the target ID exists. Doesn't it do that?

Reply:
Hmm, why so? I assumed that if there's an incorrect ID then this means that the origin element is incorrectly passed and nothing should happen. Because for drop targeting, the origin element either should not have an ID (is coming from the library for example) or should have an existing ID (an existing media is dropped into something). There shouldn't be a case where an element with incorrect ID is dropped. Thought?

Reply:
Although maybe that assumption was just in my head and it actually does already work as you're saying it should.

Will refactor a bit assuming that the ID can be passed incorrectly as well and then just the props should be taken.